salt/renderers/gpg.py

```diff
@@ -37,6 +37,56 @@
     increasing the system entropy. On virtualised Linux systems, this can often
     be achieved by installing the ``rng-tools`` package.
 
+Import keys to a master
+************************
+
+If the keys already exist and need to be imported to the salt master, run the
+following to import them.
+
+.. code-block:: bash
+
+    gpg --import /path/to/private.key
```

Comment:
the `--homedir /etc/salt/gpgkeys` part should probably be mentioned here.

Reply:
```suggestion
    gpg  --homedir /etc/salt/gpgkeys --import /path/to/private.key
```